README.md

```diff
@@ -73,48 +73,6 @@ https://github.com/webex/widgets.
 
 ### Styles
 
-In order to properly style Webex Components, we need to import all the fonts, icons, images and core _CSS_ manually.
```

Comment:
While developers won't have to import from Momentum, would they still need to import our assets in their apps?